### ISSUE TITLE
ebay-carousel: VoiceOver reads all slides aria-label at once with links and tabindex: -1

### ISSUE THREAD

**Issue by patrickufer** [eBay/evo-web#963]
I'm running into a bit of an edge-case with MacOS VoiceOver announcing every alt text for all slides, even the ones not visible in the carousel.

It announces: "list Nike Air Jordan Image 1 of 5 Nike Air Jordan Image 2 of 5 ...etc" instead of just "list Nike Air Jordan Image 1 of 5". Codex suggested setting inert on each out of view li which I tested fixes the issue. We should only add inert to hidden slides when using the carousel with a single visible slide with no peek.

I was able to replicate with some changes to the "Discrete" Storybook example. It requires 2 things:

1. Links
2. tabindex and data-carousel-tabindex -1 on those Links

https://github.com/patrickufer/evo-web/commit/480d14ec55ddea97a407a46c858f94f314565ea8

It's definitely a weird edge-case, but it's necessary for our multi-image carousel for item cards on Search.

<img width="616" height="678" alt="Image" src="https://github.com/user-attachments/assets/e56826ad-340c-451d-ba2e-73e22b82468c" />

<img width="984" height="571" alt="Image" src="https://github.com/user-attachments/assets/fd3e0a7d-50a9-429a-a5f4-5e9f3799873e" />

https://ebay-eng.slack.com/archives/CK8AX1L49/p1789587695699459
